Form-Controls/styles.css

```diff
@@ -0,0 +1,70 @@
+{
+    margin:0;
+    padding:0
+}
+.body{
+    background-position: center;
+      background-size: cover;
+      font-family: Georgia, 'Times New Roman', Times, serif;
+      margin-top: 30px;
+      font-size: 15px;
+  }
+  
+   main{
+    display: grid;
+    grid-template-columns: auto;
```

Comment:
Do you need these grid rules of auto? Take a look in Devtools at these rules and try unchecking the box next to each. What happens? Why?

https://developer.chrome.com/docs/devtools/css/issues/